reactions/src/HALO.cpp

```diff
@@ -56,8 +56,8 @@ void react_error_halo(const char* msg) {
 /* Code to calculate halo terms */
 
 /*Load the linear power spectrum */
-HALO::HALO(const Cosmology& C_, const PowerSpectrum& P_l_, real epsrel_)
-: C(C_), P_l(P_l_)
+HALO::HALO(const Cosmology& C_, const PowerSpectrum& P_l_, const PowerSpectrum& P_cb_, const PowerSpectrum& P_nu_, real epsrel_)
```

Comment:
Same as above: will this work for massless new (i.e., what if I want to recover the previous case).

Reply:
added comment. See above.